app/services/qualified_teachers_api_sender.rb

```diff
@@ -66,7 +70,7 @@ def qualified_teachers_client
   def request_body
     @request_body ||= {
       completionDate: completion_date,
-      qualificationType: participant_outcome.declaration.qualification_type,
+      qualificationType: course_short_code(participant_outcome.declaration.course),
```

Comment:
Im not sure we should tie this to the translations as they might be used elsewhere and changed 🤔